ts/sagas/startup.ts

```diff
@@ -481,7 +482,7 @@ export function* initializeApplicationSaga(
     yield* put(profileLoadSuccess(profile.value));
     yield* take(setProfileHashedFiscalCode);
   }
-
+  yield* call(setLanguageFromProfileIfExists);
```

Comment:
Should this be done only at the very first `startup`?

```suggestion
  if (!handleSessionExpiration) {
    yield* call(setLanguageFromProfileIfExists);
  }
```
What do you think about it?